### PR TITLE
feat(Yugen Anime): add presence

### LINKED ISSUE #5955: Yugenanime | yugen.to
### Website name

YugenAnime

### Website URL

https://yugen.to

### Website logo
download

### Prerequisites

- [ ] It is a paid service
- [ ] It displays NSFW content
- [ ] It is region restricted

### Description

i want the presence to show which anime am i watching , what episode am i watching. thts abt it just like zoro.to

### LINKED ISSUE #7090: Yugen Anime
### Website name

Watch HD Anime for Free - YugenAnime

### Website URL

https://yugen.to/

### Website logo

(https://yugen.to/static/img/logo__light.png)

### Prerequisites

- [ ] It is a paid service
- [ ] It displays NSFW content
- [ ] It is region restricted

### Description

Title of the anime.
What episode is being watched
timestamp

### PR DESCRIPTION
**Pull request by DiscordAnaxes**
## Description 

This pr implements, adds and or fixes the following:
- Resolves #7090 
- Resolves #5955

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary>Watching an episode</summary>

image
</details>
<details>
<summary>Viewing a series/anime</summary>

image
</details>
<details>
<summary>Viewing a anime review</summary>

image
</details>
<details>
<summary>Watching a movie</summary>

image
</details>

### CODE REVIEW

**Comment by crashbash317** [comment]
great progress so far, thank you.

File: websites/Y/Yugen Anime/presence.ts [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+		return url;
+	}
+}
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			viewHome: "general.viewHome",
+			viewAnime: "general.viewAnime",
+			watchingSeries: "general.watchingSeries",
```
Comment (DarkVillager) [inline websites/Y/Yugen Anime/presence.ts:27]: This one is not used.

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/Y/Yugen Anime/presence.ts (hunk 1 above)
Comment (DiscordAnaxes) [reply websites/Y/Yugen Anime/presence.ts:27]: I indeed do not.

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Comment by crashbash317** [comment]
image
Presence not showing anime, episode, timestamp, ect.

**Comment by DarkVillager** [comment]
> image Presence not showing anime, episode, timestamp, ect.

@DiscordAnaxes

**Comment by DiscordAnaxes** [comment]
> image Presence not showing anime, episode, timestamp, ect.

Odd it was working before. Did they update their website by any chance? Nonetheless I will fix it shortly.

**Comment by pmksch101** [comment]
> > image Presence not showing anime, episode, timestamp, ect.
> 
> Odd it was working before. Did they update their website by any chance? Nonetheless I will fix it shortly.

could you look into this please?

**Pull request merged**